src/test/fuzz/psbt.cpp

```diff
@@ -57,7 +54,7 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         (void)output.IsNull();
     }
 
-    for (size_t i = 0; i < psbt.tx->vin.size(); ++i) {
+    for (size_t i = 0; i < mtx.vin.size(); ++i) {
         CTxOut tx_out;
         if (psbt.inputs.at(i).GetUTXO(tx_out)) {
```

Comment:
It's simpler to just iterate over the `PSBTInput`s:

```suggestion
    for (const auto& input : psbt.inputs) {
        CTxOut tx_out;
        if (input.GetUTXO(tx_out)) {
```

Reply:
Thanks, fixed